.github/workflows/focal-ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: focal-ci
```

Comment:
Have you checked that this will work for the `galactic` branch even if it's merged into the `ros2` branch?

In case it only works when it's backported to `galactic`, I'd argue that it's best not to add galactic CI information to the `ros2` branch. Instead, you could give the yaml file a generic name, like `ci.yaml`, and modify it according to the branch. This also helps comparing it across branches, for example:

https://github.com/gazebosim/ros_gz/compare/galactic...ros2#diff-7c3faec84b8df09639159898858a4155d55a7febe4d37a6700f26fb6bef9826e

Reply:
I updated the PR to have a generic name `ci.yaml`, but I don't have a specific commit with the change since I rebased and squashed to add the sign-off.